GetStream/stream-chat-android
# [3496] Fix upload progress text getting ellipsized

### 🎯 Goal

The upload progress text does not have enough space to properly unfurl and often gets ellipsized.

closes #3496

### 🛠 Implementation details

- Set the visibility of `actionButton` to gone while the file is being uploaded
- Trim the padding of stream_ui_item_file_attachment.xml so that the text has more space

### 🎨 UI Changes

| Before | After |
| --- | --- |
| https://user-images.githubusercontent.com/37080097/170707652-a7a4eb24-e963-40f7-badc-f6feff052568.mp4 | https://user-images.githubusercontent.com/37080097/170707660-c17db614-9142-4550-8d48-4ff7e494c5a8.mp4 |

### 🧪 Testing

1. Upload a file
2. Observe the progress text saying " y MB / x MB"

Expected: The text should not be clipped / ellipsized.

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![progress](https://user-images.githubusercontent.com/37080097/170711358-5cd298d5-f0db-4042-a57e-cd8454bf5561.gif)
